roles/container-engine/cri-o/defaults/main.yml

```diff
@@ -5,4 +5,4 @@ crio_seccomp_profile: "/etc/crio/seccomp.json"
 
 crio_cgroup_manager: "{{ kubelet_cgroup_driver | default('cgroupfs') }}"
 
-crio_runc_path: "/usr/sbin/runc"
+crio_runc_path: "{{ runc_binary | crio_runc_path | default('/usr/sbin/runc') }}"
```

Comment:
what do you mean with `| crio_runc_path ` ???

Reply:
oh..i see `redhat.yml` should kick in but depending of ansible version ciro_runc_path may be not set (or uses the default).
better add it here:
`roles/container-engine/cri-o/vars/fedora.yml`:
```
ciro_runc_path: /usr/bin/runc
```

Reply:
Good idea!

Reply:
It looks wrong here, but I have reverted the change! (Have I?)